### PR TITLE
fresh: add Ironic Software Fresh.app v4.0.4

### PR DESCRIPTION
**Pull request by activescott**
- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by activescott** [comment]
FYI I also attempted to find a way to do a livecheck stanza but I couldn't find the necessary metadata from the developer.

**Comment by p-linnane** [comment]
Can you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap)? There's info on how to generate one in the link.

**Comment by activescott** [comment]
@p-linnane zap stanza added in cb65aed0a3. audit/style passed.

File: Casks/fresh.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 :no_check
+
+  url "https://ironicsoftware.com/downloads/Fresh.zip"
+  name "Fresh"
+  desc "Keep your recently modified files at hand and up-to-date"
+  homepage "https://ironicsoftware.com/fresh/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
```
Comment (activescott) [inline Casks/fresh.rb:12]: Interesting. Is this undocumented in the cookbooks because it "should only be used as an absolute last resort"?

Comment (miccal) [reply Casks/fresh.rb:12]: Yes -- we only use it when the `url` is un-versioned, the download size is not too large and there is no other alternative.

**Review by miccal — APPROVED** [review]

**Pull request merged**